src/core/ddsi/src/ddsi_typewrap.c

```diff
@@ -2624,7 +2622,8 @@ bool ddsi_xt_is_assignable_from (struct ddsi_domaingv *gv, const struct xt_type
     {
       case DDS_XTypes_TK_BITMASK:
         return bb == t_other->_u.bitmask.bit_bound;
-      // case TK_UINT8:   /* FIXME: TK_UINT8 not defined in idl */
+      // FIXME: TK_UINT8 not defined in idl
+      // case TK_UINT8:   
       //   return bb >= 1 && bb <= 8;
```

Comment:
@dpotman shouldn't this one be

```suggestion
      case DDS_XTypes_TK_UINT16:   
         return bb >= 1 && bb <= 8;
```

now?

Reply:
Yes, you're right (almost right, UINT8 instead of UINT16 :wink:). Apparently I missed that one when adding the type kind for UINT8, thanks for catching that!

Reply:
I'm not really good at copy-paste it seems ...

It should be separate commit (ideally a separate PR) anyway.